src/Pact/Native.hs

```diff
@@ -538,18 +538,25 @@ defineNamespaceDef = setTopLevelOnly $ defGasRNative "define-namespace" defineNa
 namespaceDef :: NativeDef
 namespaceDef = setTopLevelOnly $ defGasRNative "namespace" namespace
   (funType tTyString [("namespace", tTyString)])
-  [LitExample "(namespace 'my-namespace)"]
+  [LitExample "(namespace 'my-namespace)"
+  ,LitExample "(namespace \"\")"]
   "Set the current namespace to NAMESPACE. All expressions that occur in a current \
   \transaction will be contained in NAMESPACE, and once committed, may be accessed \
   \via their fully qualified name, which will include the namespace. Subsequent \
   \namespace calls in the same tx will set a new namespace for all declarations \
-  \until either the next namespace declaration, or the end of the tx."
+  \until either the next namespace declaration, or the end of the tx. If NAMESPACE \
+  \is the empty string, env namespace will be reset to the root space."
```

Comment:
I  realize this will have to be forked. No getting around it it we want to add this change